@pchaigno  When I'm running this command `make generate-k8s-api`in cilium root dir I'm getting below error
```
Set $GOPATH to a directory containing the Cilium repository at $GOPATH/src/github.com/cilium/cilium.
The current working directory must be the repository root for code generation to work correctly.
make: *** [Makefile:329: generate-k8s-api] Error 1
```
This is where my cilium project resides `/home/sachinmaurya/go/src/github.com/cilium`
my GOPATH is `/home/sachinmaurya/go`  ... as I can see that my cilium project is already in the dir it is mention .. I'm not getting why it is failing

--- a/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml
+++ b/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml
@@ -227,7 +227,8 @@ spec:
                     toEntities:
                       description: ToEntities is a list of special entities to which
                         the endpoint subject to the rule is allowed to initiate connections.
-                        Supported entities are `world`, `cluster` and `host`
+                        Supported entities are `world`, `cluster` ,`host` , `remote-node`,
+                        `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                       items:
                         description: Entity specifies the class of receiver/sender
                           endpoints that do not have individual identities.  Entities
@@ -1047,7 +1048,8 @@ spec:
                     toEntities:
                       description: ToEntities is a list of special entities to which
                         the endpoint subject to the rule is allowed to initiate connections.
-                        Supported entities are `world`, `cluster` and `host`
+                        Supported entities are `world`, `cluster` ,`host` , `remote-node`,
+                        `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                       items:
                         description: Entity specifies the class of receiver/sender
                           endpoints that do not have individual identities.  Entities
@@ -2580,8 +2582,8 @@ spec:
                       toEntities:
                         description: ToEntities is a list of special entities to which
                           the endpoint subject to the rule is allowed to initiate
-                          connections. Supported entities are `world`, `cluster` and
-                          `host`
+                          connections. Supported entities are `world`, `cluster` ,`host` ,
+                          `remote-node`, `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                         items:
                           description: Entity specifies the class of receiver/sender
                             endpoints that do not have individual identities.  Entities
@@ -3416,8 +3418,8 @@ spec:
                       toEntities:
                         description: ToEntities is a list of special entities to which
                           the endpoint subject to the rule is allowed to initiate
-                          connections. Supported entities are `world`, `cluster` and
-                          `host`
+                          connections. Supported entities are `world`, `cluster` ,`host` ,
+                          `remote-node`, `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                         items:
                           description: Entity specifies the class of receiver/sender
                             endpoints that do not have individual identities.  Entities

--- a/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnetworkpolicies.yaml
+++ b/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnetworkpolicies.yaml
@@ -231,7 +231,8 @@ spec:
                     toEntities:
                       description: ToEntities is a list of special entities to which
                         the endpoint subject to the rule is allowed to initiate connections.
-                        Supported entities are `world`, `cluster` and `host`
+                        Supported entities are `world`, `cluster` ,`host` , `remote-node`,
+                        `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                       items:
                         description: Entity specifies the class of receiver/sender
                           endpoints that do not have individual identities.  Entities
@@ -1051,7 +1052,8 @@ spec:
                     toEntities:
                       description: ToEntities is a list of special entities to which
                         the endpoint subject to the rule is allowed to initiate connections.
-                        Supported entities are `world`, `cluster` and `host`
+                        Supported entities are `world`, `cluster` ,`host` , `remote-node`,
+                        `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                       items:
                         description: Entity specifies the class of receiver/sender
                           endpoints that do not have individual identities.  Entities
@@ -2584,8 +2586,8 @@ spec:
                       toEntities:
                         description: ToEntities is a list of special entities to which
                           the endpoint subject to the rule is allowed to initiate
-                          connections. Supported entities are `world`, `cluster` and
-                          `host`
+                          connections. Supported entities are `world`, `cluster` ,`host` ,
+                          `remote-node`, `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                         items:
                           description: Entity specifies the class of receiver/sender
                             endpoints that do not have individual identities.  Entities
@@ -3420,8 +3422,8 @@ spec:
                       toEntities:
                         description: ToEntities is a list of special entities to which
                           the endpoint subject to the rule is allowed to initiate
-                          connections. Supported entities are `world`, `cluster` and
-                          `host`
+                          connections. Supported entities are `world`, `cluster` ,`host` ,
+                          `remote-node`, `kube-apiserver`, `init`, `health`,`unmanaged` and `all`.
                         items:
                           description: Entity specifies the class of receiver/sender
                             endpoints that do not have individual identities.  Entities

